app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,47 @@
+class OrganizationsController < ApplicationController
+  before_filter :authenticate_user!, only: [:create, :new, :edit, :update]
+
+  def index
+    @organizations = Organization.all
+  end
+
+  def show
+    @organization = OrganizationPresenter.new(Organization.find(params[:id]))
```

Comment:
Because `#show` and `#edit` use the same logic to find the organization, you could write a private method `#load_presented_record` that creates the instance variable, and then write `before_action :load_presented_record, only: [:show, :edit]` up top.